internal/safeconvert/convertuint32_arm.go

```diff
@@ -0,0 +1,18 @@
+//go:build arm || arm64
+// +build arm arm64
```

Comment:
`arm64` should not be affected by this:

```
$ export GOARCH=arm; go build  -o speaker-$GOARC ./speaker 
# go.universe.tf/metallb/internal/safeconvert
internal/safeconvert/convert.go:34:17: math.MaxUint32 (untyped int constant 4294967295) overflows int

$ export GOARCH=arm64; go build  -o speaker-$GOARC ./speaker
$ echo $?
0
```
```suggestion
// +build arm
```

Can we be more selective?

Reply:
good catch!

Reply:
fixed